praw/reddit.py

```diff
@@ -490,12 +490,14 @@ def info(
         self,
         fullnames: Optional[Iterable[str]] = None,
         url: Optional[str] = None,
+        sr_names: Optional[Iterable[str]] = None,
     ) -> Generator[Union[Subreddit, Comment, Submission], None, None]:
-        """Fetch information about each item in ``fullnames`` or from ``url``.
+        """Fetch information about each item in ``fullnames``, ``url``, or ``sr_names``.
 
         :param fullnames: A list of fullnames for comments, submissions, and/or
             subreddits.
         :param url: A url (as a string) to retrieve lists of link submissions from.
+        :param sr_names: A list of subreddit names for subreddits.
```

Comment:
Can we rename `sr_names` to `subreddits` and allow it to take in subreddit objects in addition to strings of the subreddit? That's pretty consistent with other places in PRAW.

Then one need only call `str(subreddit)` to extract the name.

Reply:
Fixed in 2931838